app/src/main/java/com/example/todoapp/viewmodel/SignInViewModel.kt

```diff
@@ -1,45 +1,41 @@
 package com.example.todoapp.viewmodel
 
 import android.util.Log
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import com.example.domain.models.SignInModel
-import com.example.domain.models.UserInfoModel
 import com.example.domain.usecases.GetCurrentUserUseCase
 import com.example.domain.usecases.SignInUseCase
+import com.example.todoapp.ui.screens.signinscreen.SignInScreenState
 import com.example.todoapp.utils.SignInUIEvent
 import dagger.hilt.android.lifecycle.HiltViewModel
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
-class SignInViewModel @Inject constructor(val signInUseCase: SignInUseCase ,
-  val getCurrentUserUseCase: GetCurrentUserUseCase) :
+class SignInViewModel @Inject constructor(
+  val signInUseCase: SignInUseCase,
+  val getCurrentUserUseCase: GetCurrentUserUseCase
```

Comment:
As we are overriding the invoke operator and using classes as it's functions, you can remove useCase from the end of variables names
```suggestion
  val signIn: SignInUseCase,
  val getCurrentUser: GetCurrentUserUseCase
```